src/CommandHandler/Wishlist/AddSelectedProductsToCartHandler.php

```diff
@@ -79,6 +83,16 @@ private function isInStock(WishlistItem $wishlistProduct): bool
         return false;
     }
 
+    private function productHasPositiveQuantity(OrderItemInterface $product): bool
```

Comment:
Could you extract it into checker class and spec?

Reply:
Done.